peps/pep-0772.rst

```diff
@@ -184,201 +184,196 @@ making process that governs the technical projects under the PyPA umbrella.
+a Council member changing employment), then one or more Council members must resign to remedy the issue, and
+the resulting vacancies can then be filled as `normal <vacancy>`_.
 
---------------
-Voting members
---------------
+.. _electors:
 
-Responsibility
-==============
+==========================
+Packaging Council Electors
+==========================
+
+----------------
+Responsibilities
+----------------
+
+Packaging Council Electors participate in formal votes to elect the Packaging Council.
 
-Voting members participate in formal votes to elect the Packaging Council.
+The eligibility of Packaging Council Electors is equivalent to the Article IV, section 4.2 voting membership
+defined in the `PSF Bylaws <https://www.python.org/psf/bylaws/>`_.  Should those bylaws change in the future,
+the eligibility of Packaging Council Electors will similar change to match.  As with PSF voting membership,
+Packaging Council Electors must affirm their intention to vote in Packaging Council elections every year.
 
 .. _process:
 
 Processes
 =========
 
-Initial membership
-------------------
-
-Initial set of voting members will include anyone who has taken the time to
-formalise their participation in the packaging community. This includes:
-
-* PyPA members: Anyone with the triage bit or commit bit, or at least one
-  project in the PyPA organisation.
-* Packaging workgroup members: Anyone who is listed on the Packaging WG charter
-  who is willing to participate is welcome.
-* Interested core team members: Any Python core team member who is willing to
-  participate is welcome.
-* Wider community members: An initial set of for-profit companies, nonprofit
-  organizations, academic or educational institutions and smaller unaffiliated
-  projects would be invited to nominate three individuals to represent them.
-  See :ref:`the initial set of community voting members <appendix_C>`.
-
-Adding a new member
--------------------
-
-New voting members are added by a simple majority vote by the current
-membership. Quorum for adding new members is 50%.
-
-A vote to add a new member is triggered when a voting member calls for one
-publicly on an appropriate communication channel, and another voting member
-seconds the call within two weeks.
-
-The vote lasts for one week. Each member can vote for or against.
-
 Removal of a member
 -------------------
 
-In order to maintain a reasonable expectation of quorum, failure to participate
-in Packaging Council elections for two consecutive council elections
-automatically removes a person from the list of voting members, until they
-re-submit their intention to resume their participation to the Packaging
-Council in writing.
+In order to maintain a reasonable expectation of quorum, failure to participate in Packaging Council elections
+for two consecutive council elections automatically removes a person from the list of Packaging Council
```

Comment:
This was made uppercase elsewhere:
```suggestion
for two consecutive Council elections automatically removes a person from the list of Packaging Council
```

Reply:
Wrapping was a struggle actually.  I noticed some wrapping bugs in my IDE and just didn't want to shave the yak.